static/js/util.js

```diff
@@ -254,6 +254,47 @@ exports.prefix_sort = function (query, objs, get_item) {
              rest:    noMatch };
 };
 
+exports.emoji_prefix_sort = function (query, objs, get_item) {
+    // Similar to prefix sort above, but takes into account popular emojis
+    var popular_emojis = [];
+    var beginswithCaseSensitive = [];
+    var beginswithCaseInsensitive = [];
+    var noMatch = [];
+    var obj;
+    var item;
+    var frequently_used_emojis_list = [
```

Comment:
It would be good not to repeat this list; it must appear somewhere else as well?

Reply:
In the first commit, you can see that I'm accessing this list from `emoji_picker.js`. But when I ran the test `test-js-with-node` it fails with a message `ReferenceError: emoji_picker is not defined`, which is why I made the second commit. On my local dev environment however, the first commit works just fine, so I'm not sure why the test failed for the first commit

Reply:
good question. Worth asking in "#development help" how to make this work.